File: app/src/main/java/com/jossy/android/mobilebenchmarkappjava/service/ApiService.java
package com.jossy.android.mobilebenchmarkappjava.service;

import com.jossy.android.mobilebenchmarkappjava.data.Post;

import java.util.List;
import retrofit2.Call;
import retrofit2.http.GET;

public interface ApiService {
    @GET("posts")
    Call<List<Post>> getPosts();
}

File: kotlin/app/src/main/kotlin/com/jossy/android/mobilebenchmarkappkotlin/services/ApiService.kt
package com.jossy.android.mobilebenchmarkappkotlin.service

import com.jossy.android.mobilebenchmarkappkotlin.data.Post
import retrofit2.Call
import retrofit2.http.GET

interface ApiService {
    @GET("posts")
    fun getPosts(): Call<List<Post>>
}
